managers/static/managers/scripts/expected_hours_caleandar_ajax.js

```diff
@@ -0,0 +1,137 @@
+const projectDataSelectOptions = document.getElementById('project-select-options')
+const projectSelect = document.getElementById('project-select')
+const employeeDataListOptions = document.getElementById('employee-list-options')
+const employeeSelect = document.getElementById('employee-list')
+const calendarList = document.getElementById('calendar-feature')
+const calendarSettings={
```

Comment:
missing spaces between equal mark and bracet

Reply:
Changed